composer.json

```diff
@@ -28,6 +28,7 @@
         "aic/data-hub-foundation": "dev-laravel-8-support",
         "aic/laravel-scout-elastic": "dev-master",
         "cviebrock/laravel-elasticsearch": "^9.0",
+        "doctrine/dbal": "^2.13",
```

Comment:
@zachgarwood Are you using this package anywhere in this commit?

Reply:
Yeah, it was necessary to install it in order to run a migration that altered a `TIMESTAMP` column without throwing an error. See the database config: https://github.com/art-institute-of-chicago/data-aggregator/pull/56/files#diff-e4382b565f69d02de16eef89c8d5ca2b60a679ffca90ab8b7d85fbd78f30075b. I don't believe this issue showed up before because the testing database was sqlite, which doesn't have strict column types.